scripts/vcf_site_level_filtering_wdl/JointVcfFiltering.wdl

```diff
@@ -177,10 +181,11 @@ task ExtractVariantAnnotations {
 		File extracted_training_vcf = "~{basename}.~{mode}.vcf.gz"
 		File extracted_training_vcf_index = "~{basename}.~{mode}.vcf.gz.tbi"
 		Array[File] outputs = glob("~{basename}.~{mode}.*")
+		File monitoring_log = "monitoring.log"
 	}
 	runtime {
 		docker: gatk_docker
-		disks: "local-disk " + disk_size + " LOCAL"
```

Comment:
Interesting thing here - 'LOCAL' calls for SSD in chunks of 375GB https://cloud.google.com/compute/docs/disks/#overview (as I read it). I switched to HDD for cost. I *don't* think performance took a big hit